var/spack/repos/builtin/packages/plplot/package.py

```diff
@@ -101,6 +87,32 @@ def cmake_args(self):
         else:
             args += ['-DENABLE_qt=OFF']
 
+        if '+tcl' in self.spec:
+            args += ['-DENABLE_tcl=ON']
+            # could also be addressed by creating the links within tcl
+            # as is done for the tclsh executable
+            args += [
+                '-DTCL_INCLUDE_PATH={0}/include'.format(
+                    self.spec['tcl'].prefix.include
+                ),
+                '-DTCL_LIBRARY={0}'.format(
+                    LibraryList(find_libraries(
+                        'libtcl*',
```

Comment:
Won't this also pick up `libtclstub`?

Reply:
`libtclstub` is a static library, `libtcl` a dynamic one.
It worked fine in my tests (but I can't say it will work fine in every case).